libr/main/radare2.c

```diff
@@ -399,6 +399,17 @@ static bool mustSaveHistory(RConfig *c) {
 
 // Try to set the correct scr.color for the current terminal.
 static void set_color_default(void) {
+#ifdef __WINDOWS__
+	char *alacritty = r_sys_getenv ("ALACRITTY_LOG");
```

Comment:
thats leaking, you should free that alacritty thing

Reply:
sorry didn't realize this